pkg/diff/diff.go

```diff
@@ -1060,6 +1061,43 @@ func HideSecretData(target *unstructured.Unstructured, live *unstructured.Unstru
 			}
 		}
 	}
+
+	// hide annotations
+	for _, k := range hideAnnots {
```

Comment:
can we move this part to generic function that will hide some values, only differences between your change and the secrets hiding that already exists. 

Only difference is how you fetch "data". I think it can be generic function that accept another function that return "data"

Reply:
Incorporated your suggestion. Moved hide logic to a generic function.

Reply:
@svghadi will review it tomorrow